sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/Aggregation.java

```diff
@@ -67,4 +68,26 @@ static Aggregation explicitBucketHistogram() {
   static Aggregation explicitBucketHistogram(List<Double> bucketBoundaries) {
     return ExplicitBucketHistogramAggregation.create(bucketBoundaries);
   }
+
+  /**
+   * Aggregates measurements into a base-2 {@link MetricDataType#EXPONENTIAL_HISTOGRAM} using the
+   * default {@code maxBuckets} and {@code maxScale}.
+   */
+  static Aggregation base2ExponentialBucketHistogram() {
```

Comment:
The name was switched to "base2 exponential bucket histogram" to avoid confusing folks that may expect "exponential" to refer to base 10. Convo available [here](https://github.com/open-telemetry/opentelemetry-specification/pull/3041#discussion_r1066038209).

Reply:
Oh, and I'm planning on adding `Base2` prefix to the internal `ExponentialHistogram*` classes in a followup.